api/envoy/config/trace/v3/datadog.proto

```diff
@@ -16,6 +18,13 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Datadog tracer]
 
+// Configuration for the Remote Configuration feature.
+message DatadogRemoteConfig {
+  // Frequency at which new configuration updates are queried.
+  // [#comment: If no value is provided, the default value is delegated to the Datadog tracing library.]
```

Comment:
The comment tag will remove it from the docs. We want this stuff to show up in the docs. Same below.

Reply:
Good to know. Resolved in 006768b7b97f2adb4d40ad5d9f48ade0d58888fe